Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/FareCalculator/Calculator.hs

```diff
@@ -106,7 +106,7 @@ fareSum fareParams = case fareParams.farePolicyType of
 
 normalFareSum :: FareParameters -> Money
 normalFareSum fareParams = do
-  baseFareSum fareParams + (fromMaybe 0 fareParams.deadKmFare) + fromMaybe 0 fareParams.driverSelectedFare + fromMaybe 0 fareParams.customerExtraFee
+  baseFareSum fareParams + (fromMaybe 0 fareParams.deadKmFare) + fromMaybe 0 fareParams.driverSelectedFare
 
 slabFareSum :: FareParameters -> Money
 slabFareSum = baseFareSum
```

Comment:
Why logic for slab is so different from normal one? Why every part of slab fare considered as part of base fare? Why `fareParams.nightCoefIncluded` used in calculations directly, without `calculateDayPartRate`? @hkmangla Could you check this logic, is it correct?

Reply:
Slab fare policy changes was done in very short notice, so this code is not clean, just working anyhow. @khuzema786 is working on cleaning this. @DonDokidon We can review his PR and suggest more refactoring if required.